src/app/containers/card-deck/card-deck.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, OnInit, Renderer2 } from '@angular/core';
+import { MonsterCard } from '../../models/card.model';
+
+@Component({
+    selector: 'card-deck',
+    styleUrls: ['card-deck.component.scss'],
+    template: `
+        <div *ngFor="let c of monsterCards" class="card-container">
+            <monster-card (click)="cardClick(c)"
+                [ngClass]="{
+                    'flipped flip-to-front': c.facing === 'up',
+                    'send-to-back': c.facing === 'down',
+                    'shuffle': this.shuffleDeck === true
```

Comment:
would it be `this.shuffleDeck` or `shuffleDeck`?

Reply:
N/A, but honestly I'm not sure